query.go

```diff
@@ -369,6 +369,56 @@ func labelFilter(target string, useSep bool) driver.FilterFn {
 	})
 }
 
+func GetName(info driver.Info) (name string, id string) {
+	nameParts := strings.Split(info.Name, camera.LabelSeparator)
+	if len(nameParts) > 1 {
+		name, id = nameParts[0], nameParts[1]
+	} else {
+		// fallback to the label if the name does not have an any additional parts to use.
+		name, id = nameParts[0], info.Label
+	}
+
+	name, id = strings.TrimSpace(name), strings.TrimSpace(id)
+	return
+}
+
+func parseNameAndId(toParse string) (name string, id string) {
+	// expects '<camera_name> (<camera_id>)'
```

Comment:
This works but depends on the front-end presenting the name this way. If this is approved I'll leave a comment stating this in the front-end so we don't inadvertently break anything.